playbooks/openshift/roles/common/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: reload firewalld
```

Comment:
Agreed. It worked perfectly fine on my system here. When I moved it to CentOS 7.2 on the slave, it started misbehaving. I am using 1.9.6 during this point, so the invocation and behavior might be different, or a bug. I could go back to the virtualenv, I suppose, but that's another configuration beyond what I'm currently doing.